src/test-explorer/resolver.ts

```diff
@@ -256,6 +261,7 @@ export class TestResolver implements OnModuleInit, vscode.Disposable {
       await this.expandTargetsForDocument(doc)
     }
 
+    if (this.openDocumentWatcherEnabled) return
```

Comment:
So this will always bail?

Reply:
I guess in this case it will actually never bail because it starts off as false.

I meant to use this to set the watcher on only the first run, I have updated with the missing line (265) to handle this.